en-US/Punctuation.xml

```diff
@@ -107,15 +107,53 @@
 							magenta
 						</para>
 
-					</listitem>
+					</listitem> 
 
-				</itemizedlist>
+				</itemizedlist> 
+
+			</listitem>  -->
+
+		</itemizedlist> 
+
+		 <para>
+			Use a semicolon to separate items in a series if the items contain commas.
+		</para>
+		 <itemizedlist>
+			<listitem>
+				<para>
+					Everyday I have coffee, toast, and fruit for breakfast; a salad for lunch; and a peanut butter sandwich, cookies, ice cream, and chocolate cake for dinner.
```

Comment:
```suggestion
					Every day I have coffee, toast, and fruit for breakfast; a salad for lunch; and a peanut butter sandwich, cookies, ice cream, and chocolate cake for dinner.
```

Reply:
"Everyday" is the adjective form meaning "used or seen daily" or "ordinary". "Every day" is the adverb phrase meaning "daily" or "every weekday". (Paraphrasing Merriam-Webster.)